pkg/reconciler/route/resources/filters.go

```diff
@@ -28,8 +28,7 @@ type Filter func(service *corev1.Service) bool
 func FilterService(services []*corev1.Service, acceptFilter Filter) []*corev1.Service {
 	var filteredServices []*corev1.Service
```

Comment:
I was wondering if we could init a capacity for FilterService... but I can't find references? Is this code used?

Reply:
In fact you're right. Let me remove it.